### DISCUSSION TITLE
所有pan.linuxgame.cn 网页无法打开

### DISCUSSION THREAD

**Discussion by fantong4** [Gamuxorg/bbs#305] [游戏或网站运行反馈]
以下是我点开下载链接后得到的结果

This site can’t be reached
pan.linuxgame.cn’s server IP address could not be found.
DNS_PROBE_FINISHED_NXDOMAIN

虽然linuxgame.cn 这个网站本身可以打开， 但是在所有游戏的游戏下载那一栏游戏链接的容量都显示为null

**Comment by wangling12** [comment]
我们更换了下载方式。下载连接还没来得及更新。你可以先去[这里](https://de.linuxgame.cn)下载。

**Comment by shouhuanxiaoji** [comment]
已解决
